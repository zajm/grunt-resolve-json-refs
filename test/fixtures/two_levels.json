{
    "A": "Included",
    "B": "@:A",
    "C": "@:B"
}