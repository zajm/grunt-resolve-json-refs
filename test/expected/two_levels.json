{
    "A": "Included",
    "B": "Included",
    "C": "Included"
}